README.md

```diff
@@ -211,34 +261,34 @@ The inputs for this request will be:
 
 - *Prefix* : prefix where we want to create the children under. This argument will be
 extracted from the URL ```/child-barcodes/<PREFIX>/new```
-All barcodes for the children will have this prefix (example, prefix HT will generate children
-like HT-11111-1, HT-11111-2, etc)
+All barcodes for the children will have this prefix (example, prefix SS will generate children
+like SS-11111-1-L, SS-11111-2-M, etc)
 - *Parent Barcode* : barcode that will act as parent of the children. This argument will be
-extracted from the Body of the request, eg: ```{'barcode': 'HT-1-1', 'count': 2}```.
-To be considered valid, the barcode needs to follow the format ```<PREFIX>-<NUMBER>(-<NUMBER>)?```
-where the last number part is optional (it represents if the barcode was a child).
-For example, valid barcodes would be ```HT-11111-13``` (normal parent) and ```HT-11112-24```
-(parent that was a child) but not ```HT-1-1-1``` or ```HT12341-1```.
+extracted from the Body of the request, eg: ```{'barcode': 'SS-1-1-L', 'count': 2}```.
+To be considered valid, the barcode needs to follow the format ```<PREFIX>-<NUMBER>(-<NUMBER>)?(-<CHECKSUM>)```
+where the second number part is optional and it represents if the barcode was a child.
+For example, valid barcodes would be ```SS-11111-13-M``` (normal parent) and ```SS-11112-24-N```
+(parent that was a child) but not ```SS-1-1-1-L``` (several '-') or ```SS12341-1-L``` (no '-' separation).
 - *Child* : part of the Parent barcode string that would identify if the parent was
-a child before (the last number). For example for the *Parent barcode* ```HT-11111-14```,
-*Child* would be 14; but for the *Parent barcode* ```HT-11111```, *Child* would have no
+a child before (the last number). For example for the *Parent barcode* ```SS-11111-14-R```,
+*Child* would be 14; but for the *Parent barcode* ```SS-11111-W```, *Child* would have no
 value defined.
 - *Count* : number of children barcodes to create.
 
 #### Wrong parent barcodes
 
 A request with parent barcode that does not follow the format defined like:
 ```
-<PREFIX>-<NUMBER>(-<NUMBER>)?
+<PREFIX>-<NUMBER>(-<NUMBER>)?(-<CHECKSUM>)
```

Comment:
Is this the same for all formats?

Reply:
Only for formats that accept children barcodes